napi/Cargo.toml

```diff
@@ -10,11 +10,21 @@ repository = "https://github.com/napi-rs/napi-rs"
 version = "0.5.1"
 
 [features]
+default = [ "napi7" ]
+
 latin1 = ["encoding_rs"]
 libuv = ["futures"]
 serde-json = ["serde", "serde_json"]
 tokio_rt = ["futures", "tokio", "once_cell"]
 
+napi7 = [ "napi6" ]
```

Comment:
what if add a feature `system-napi` to compatible with the previous behavior? So that you don't need to change the build scripts on CI and people have more choice.

Reply:
I wanted to avoid that that since thats probably not what most people want, but I agree that it would simplify everything greatly and provide backwards compatibility. I'll make that change and document these features on the readme.